tiled/catalog/adapter.py

```diff
@@ -1462,7 +1462,7 @@ def structure_family(query, tree):
 CatalogNodeAdapter.register_query(Contains, contains)
 CatalogNodeAdapter.register_query(In, partial(in_or_not_in, method="in_"))
 CatalogNodeAdapter.register_query(NotIn, partial(in_or_not_in, method="not_in"))
-CatalogNodeAdapter.register_query(KeyExists, has_key)
+CatalogNodeAdapter.register_query(KeyPresent, has_key)
```

Comment:
Can we rename this function to `key_present` just to make it easier to find? The implementation names generally match the query names.